tests/test_authcode.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import patch
```

Comment:
Azure CLI as an application can move on to use newer version of Python, but MSAL as a library still tries to keep compatibility with older Python when possible.

Here, `mock` is not a built-in module in Python 2.7, so the [build actions are failing](https://github.com/AzureAD/microsoft-authentication-library-for-python/actions). I'd suggest either declare a test dependency of external `mock` when running on Python 2.7, or simply remove this test case. Seriously, for logic as simple as this pr, I do not think we have to rely on test case like this.